blog/2022-08-21-welcome/index.md

```diff
@@ -0,0 +1,10 @@
+---
+slug: welcome
+title: Welcome
+authors: [afflom, jpower432, sabre1041, usrbinkat]
+tags: [facebook, hello, docusaurus]
```

Comment:
Remove

Reply:
resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)